src/main/java/hunternif/mc/atlas/item/ItemAtlas.java

```diff
@@ -22,7 +22,7 @@
 	static final String WORLD_ATLAS_DATA_ID = "aAtlas";
 
 	public ItemAtlas() {
-	    setUnlocalizedName("antiqueAtlas");
+	    setTranslationKey("antiqueAtlas");

```

Comment:
Is this an MCP mapping change?

Reply:
Yes, moved to stable_39 as it's the final 1.12.2 mapping.